src/renderer/renderElement.ts

```diff
@@ -426,6 +426,9 @@ export function renderElementToSvg(
             offsetY || 0
           }) rotate(${degree} ${cx} ${cy})`,
         );
+        if (element.type === "line") {
```

Comment:
Line is very common, is it possible to only add this attribute to lines where the shape is closed and there’s a background to avoid bloating the exported svg?

Reply:
Fair point. I was too eager to merge because it fixed a bug.  
@lusingander could you add a conditional check here, if not I can create a quick PR

Reply:
@vjeux @pshihn 
Thanks for the comment, I'll work it.
Since it doesn't seem to have the `closed` property, should we use the `isPathALoop` to check it?

Reply:
Could you confirm #1397?